### ISSUE TITLE
Accessibility - Multistep dialog missing dark mode styles in step chooser

### ISSUE THREAD

**Issue by benrucker** [palantir/blueprint#7505]
I noticed that the step chooser LHP in the multistep dialog doesn't look right in dark mode. The text of an active step is using `blue3` when I would expect it to be `blue5`. This causes very low contrast in dark mode and some visual jitter to me.
